frontend/wallet/src/render/components/TransactionsView.re

```diff
@@ -34,81 +34,61 @@ module Styles = {
     ]);
 };
 
-let myWallets = [PublicKey.ofStringExn("PUB_KEY_E9873DF4453213303DA61F2")];
-let otherKey = PublicKey.ofStringExn("BDK342322");
+module Transactions = [%graphql
+  {|
+    query transactions($publicKey: String!) {
+      blocks(filter: { relatedTo: $publicKey }) {
+        nodes {
+          creator @bsDecoder(fn: "Apollo.Decoders.publicKey")
+          protocolState {
+            blockchainState {
+              date @bsDecoder(fn: "Apollo.Decoders.date")
+            }
+          }
+          transactions {
+            userCommands {
+              to_: to @bsDecoder(fn: "Apollo.Decoders.publicKey")
+              from @bsDecoder(fn: "Apollo.Decoders.publicKey")
+              amount @bsDecoder(fn: "Apollo.Decoders.int64")
+              fee @bsDecoder(fn: "Apollo.Decoders.int64")
+              memo
+              isDelegation
+              date @bsDecoder(fn: "Apollo.Decoders.date")
+            }
+            feeTransfer {
```

Comment:
ah I see you did the `.` thing so this would auto coerce -- I think we should use the `@bsRecord` here in that case as we have the record definition above already https://github.com/apollographql/reason-apollo#access-deeply-nested-optional-objects

In general, for the things we map into a ViewModel anyway, we can leave as objects, but for things that are going directly to reason I believe we should go for the bsRecord where possible.

Reply:
oh yeah bsRecord + bsDecoder might work well together, nice

Reply:
We can leave this for later